addons/mass_mailing/models/mailing.py

```diff
@@ -208,6 +208,7 @@ def _compute_clicks_ratio(self):
             FROM mailing_trace AS stats
             LEFT OUTER JOIN link_tracker_click AS clicks ON clicks.mailing_trace_id = stats.id
             WHERE stats.mass_mailing_id IN %s
+            AND stats.trace_status not in ('bounce', 'cancel', 'error')
```

Comment:
Quickly looking at code, it seems other ratio are computed using "total = row['expected'] - row['canceled']"; here we remove bounced and error also. It does not seem coherent ?